src/js/core/widget/core/Slider.js

```diff
@@ -715,88 +723,138 @@
 				self._previousValue = self.element.value;
 			};
 
+			/**
+			 * touchstart event handler
+			 * @method _onTouchStart
+			 * @member ns.widget.core.Slider
+			 * @protected
+			 */
 			prototype._onTouchStart = function () {
 				this._ui.handlerElement.classList.add(classes.SLIDER_HANDLER_ACTIVE);
 			};
 
+			/**
+			 * touchend event handler
+			 * @method _onTouchEnd
+			 * @member ns.widget.core.Slider
+			 * @protected
+			 */
 			prototype._onTouchEnd = function () {
 				this._ui.handlerElement.classList.remove(classes.SLIDER_HANDLER_ACTIVE);
 			};
 
-			// prototype._onFocus = function () {
-			// 	var container = this._ui.barElement.parentElement;
-
-			// 	container && container.classList.add("ui-listview-item-focus");
-			// };
-
-			// prototype._onBlur = function () {
-			// 	var container = this._ui.barElement.parentElement;
-
-			// 	container && container.classList.remove("ui-listview-item-focus");
-			// };
-
+			/**
+			 * Decrease slider current value by one step (triggers only input event)
+			 * @method _decreaseValue
+			 * @member ns.widget.core.Slider
+			 * @protected
+			 */
 			prototype._decreaseValue = function () {
 				var self = this;
 
 				self._setValue(self._value - (parseFloat(self.element.step) || 1));
+				events.trigger(self.element, "input");
 			};
 
+			/**
+			 * Increase slider current value by one step (triggers only input event)
+			 * @method _increaseValue
+			 * @member ns.widget.core.Slider
+			 * @protected
+			 */
 			prototype._increaseValue = function () {
 				var self = this;
 
 				self._setValue(self._value + (parseFloat(self.element.step) || 1));
+				events.trigger(self.element, "input");
 			};
 
-			// prototype._lockKeyboard = function () {
```

Comment:
I cannot verify this change during this review.

Reply:
I wonder if we should remove this code

Reply:
This code was used to lock keyboard support inside selected slider.
For now this is mechanism is handled in different way. I have added to base keyboard support key event delegation if any of the widgets after "actionEnter" will capture events.
Also disabling event delegation is now handled by BaseKeyboardSupport not by widget itself.

I think we can safely remove this code.